pythagorean_thm.py

```diff
@@ -0,0 +1,27 @@
+'''
+This script will help solve the pythagorean theorem.
+'''
+
+import math
+
+def main(a, b, c):
```

Comment:
What if all 3 inputs are none or all 3 inputs are not-none?
Recommend adding a check to ensure exactly one argument is none.